plugins/woocommerce/src/StoreApi/Schemas/V1/BillingAddressSchema.php

```diff
@@ -145,7 +144,15 @@ function( $carry, $key ) use ( $additional_address_fields ) {
 				$address_object[ $field ] = '';
 			}
 
-			return $this->prepare_html_response( $address_object );
+			foreach ( $address_object as $key => $value ) {
+				if ( 'boolean' === $this->get_properties()[ $key ]['type'] ) {
```

Comment:
Is `$this->get_properties()[ $key ]` always guaranteed to be there? Could we add a check first to prevent warnings?